flow/layers/backdrop_filter_layer_unittests.cc

```diff
@@ -52,19 +54,19 @@ TEST_F(BackdropFilterLayerTest, EmptyFilter) {
   layer->Add(mock_layer);
 
   layer->Preroll(preroll_context(), initial_transform);
-  EXPECT_EQ(layer->paint_bounds(), child_bounds);
+  EXPECT_EQ(layer->paint_bounds(), preroll_context()->cull_rect);
```

Comment:
In general, we try to test for specific answers rather than to side effect them from other structures that participate in the test. Testing this against kGiantRect would keep with that goal better. (Repeat comment for other tests below...)